recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston

```diff
@@ -384,14 +384,6 @@ DISTRO_FEATURES_append = " ivi-shell"
 # Add Capacity Aware migration Strategy (CAS)
 MACHINE_FEATURES_append = " cas"
 
-# U-boot/IPL option for H3 (SoC: r8a7795)
-# For H3 SiP DDR 4GiB (1GiB x 4ch)
-#H3_OPTION = "0"
-# For H3 SiP DDR 8GiB (2GiB x 4ch)
-H3_OPTION = "1"
-# For H3 SiP DDR 4GiB (2GiB x 2ch)
-#H3_OPTION = "2"
-
```

Comment:
please remove this with a proper reverting the hack commit

Reply:
Done

Reply:
You missed a signed-off in a revert commit.